apps/site/src/app/apply/sections/Form/ResumeOutputFeedback.tsx

```diff
@@ -0,0 +1,20 @@
+interface outputFeedbackProps {
+	errorMessage: string;
+	resumePath: string;
+}
+
+export default function OutputFeedBack(props: outputFeedbackProps) {
```

Comment:
Suggestion: you can extract the properties out from the props like so: `OutputFeedback({errorMessage, resumePath}: OutputFeedbackProps)`